### ISSUE TITLE
Only supports python 3.10

### ISSUE THREAD

**Issue by GJBoth** [TuragaLab/flybody#1]
Installing using python 3.11 gives an error - we should limit it to 3.10 only.

**Comment by vaxenburg** [comment]
Right, that's because the test only tests the [core installation](https://github.com/TuragaLab/flybody/tree/main?tab=readme-ov-file#option-1-installation-from-cloned-local-repo)
